tests/lora/utils.py

```diff
@@ -690,14 +708,21 @@ def test_simple_inference_with_text_denoiser_lora_and_scale(self):
         scheduler_classes = (
             [FlowMatchEulerDiscreteScheduler] if self.uses_flow_matching else [DDIMScheduler, LCMScheduler]
         )
+        call_signature_keys = inspect.signature(self.pipeline_class.__call__).parameters.keys()
```

Comment:
this can be a property (child class can overwrite it too) - ok to keep it as it is the test here;